atom/browser/ui/tray_icon_cocoa.mm

```diff
@@ -30,6 +30,7 @@ @interface StatusItemView : NSView {
   base::scoped_nsobject<NSImage> alternateImage_;
   base::scoped_nsobject<NSString> title_;
   base::scoped_nsobject<NSStatusItem> statusItem_;
+  NSTrackingArea *trackingArea_;
```

Comment:
This should be a managed pointer:

`base::scoped_nsobject<NSTrackingArea> trackingArea_;`